sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveMetastoreCatalogSuite.scala

```diff
@@ -363,4 +363,33 @@ class DataSourceWithHiveMetastoreCatalogSuite
       }
     })
   }
+
+  Seq(
+    "parquet" -> "org.apache.hadoop.hive.ql.io.parquet.serde.ParquetHiveSerDe",
+    "orc" -> "org.apache.hadoop.hive.ql.io.orc.OrcSerde"
+  ).foreach { case (format, serde) =>
+    test("SPARK-28266: convertToLogicalRelation should not interpret `path` property when " +
+      s"reading Hive tables using $format file format") {
+      withTempPath(dir => {
+        val baseDir = s"${dir.getCanonicalFile.toURI.toString}"
```

Comment:
Seems that the string interpolation is unnecessary here?

Also can we just use `dir.getAbsolutePath`? Or do we require a URI here?

Reply:
Fixed! Copied it from a test above, didn't pay too much attention.